backend/src/schema/resolvers/groups.ts

```diff
@@ -11,6 +11,8 @@ import Resolver, {
 import { mergeImage } from './images/images'
 import { createOrUpdateLocations } from './users/location'
 
+// Wolle console.log('DESCRIPTION_WITHOUT_HTML_LENGTH_MIN: ', DESCRIPTION_WITHOUT_HTML_LENGTH_MIN)
+
```

Comment:
```suggestion
```

Reply:
I's a work in progress comment. I'll remove it later.